SharpGenTools.Sdk/SharpGenTools.Sdk.targets

```diff
@@ -90,13 +115,10 @@
     Outputs="@(TfmConsumerProps)"
      Condition="'$(SharpGenGenerateConsumerBindMapping)' == 'true'"
   >
-    <PropertyGroup>
-      <EmbeddedConsumerProp>$</EmbeddedConsumerProp>
-    </PropertyGroup>
     <ItemGroup>
       <TfmConsumerPropsLines Include="&lt;Project&gt;" />
       <TfmConsumerPropsLines Include="&lt;ItemGroup&gt;" />
-      <TfmConsumerPropsLines Include='&lt;SharpGenConsumerMapping Include=&quot;$(EmbeddedConsumerProp)(MSBuildThisFileDirectory)/$(SharpGenConsumerBindMappingConfigId).BindMapping.xml&quot;/&gt;' />
+      <TfmConsumerPropsLines Include='&lt;SharpGenConsumerMapping Include=&quot;%24(MSBuildThisFileDirectory)/$(SharpGenConsumerBindMappingConfigId).BindMapping.xml&quot;/&gt;' />
```

Comment:
Did you validate that the `%24` works? I thought that that didn't work and you'd have to use something like the built-in `MSBuild::Escape` method.

Reply:
Yes, I've validated this to work.